packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testTruncate.st

```diff
@@ -0,0 +1,9 @@
+testing
+testTruncate
+
+	| scrollPane |
+	instance bounds: (50 @ 50 corner: 200 @ 200).
+	scrollPane := SScrollPane forMorph: instance.
+	instance truncateDiagram.
+
+	self assert: (instance boundsInWorld containsRect: scrollPane boundsInWorld).
```

Comment:
yes, but unfortunately the test doesn´t really work yet. im currently thinking of another way to test it.